project/src/main/java/com/google/sps/data/CloudStorageLibrary.java

```diff
@@ -33,21 +31,27 @@
 import com.google.cloud.storage.Storage;
 import com.google.cloud.storage.StorageClass;
 import com.google.cloud.storage.StorageOptions;
+import com.google.cloud.storage.Blob;
 
 public class CloudStorageLibrary {
 
     //Function to determine if the gcs file path is valid
-    public static Boolean doesGcsuriExist(BlobstoreService blobstore, BlobInfoFactory blobInfoFactory,
-                                           String gcsFilePath) {
-        BlobKey blobKey = blobstore.createGsBlobKey(gcsFilePath);
-        BlobInfo blobInfo = blobInfoFactory.loadBlobInfo(blobKey);
+    public static Boolean doesGcsuriExist(Storage storage, String gcsFilePath) {
+        String fileName = gcsFilePath.replaceFirst("/gs/cloudberry-step-2020-test-bucket/", "");
+        Blob blob = storage.get("cloudberry-step-2020-test-bucket", fileName);
```

Comment:
You could have this constant as like a private static final in the library, and just call that variable here instead